### ISSUE TITLE
[BUG] This may be due to their app version or other settings

### ISSUE THREAD

**Issue by nebula0225** [adw0rd/instagrapi#1240]
When send direct message
this error showed up 

`private_request - INFO - username [403] POST https://i.instagram.com/api/v1/direct_v2/threads/broadcast/link/ (ip, OnePlus 6T Dev)`

`{"action":"item_ack","status_code":"403","payload":{"client_context":"","message":"This may be due to their app version or other settings. Client ``context:"},"status":"fail"}`

**Comment by nashsophie** [comment]
me too

**Comment by adw0rd** [comment]
Submit your PR with fixes please

**Comment by jiangcheng1214** [comment]
To me, I hit this issue only when I send DM to certain users while sending DM to other users was fine.

**Comment by inchargeads** [comment]
@jiangcheng1214 Yep same, I think the solution is finding a better way to validate if it's okay to send a message to a user before sending.

**Comment by Ashad001** [comment]
> @jiangcheng1214 Yep same, I think the solution is finding a better way to validate if it's okay to send a message to a user before sending.

@inchargeads is there a way to validate..?

**Comment by subzeroid** [comment]
Closing this old Direct link-share report as not actionable without a current reproducible app/library mismatch.

The 403 response text from Instagram says the recipient cannot receive that message because of app version or account settings. That is a server-side delivery restriction for the target recipient, not enough by itself to identify a library payload bug.

If the current Instagram app can send the same link to the same recipient but the current instagrapi release still receives this 403, please open a fresh issue with the current version, the exact helper used, and a sanitized request/response comparison. That would give us enough evidence to fix the payload rather than guess.
